### ISSUE TITLE
*Non-breaking space* used instead of *space*

### ISSUE THREAD

**Issue by thiagola92** [dialogic-godot/documentation#36]
Some spaces got replaced by [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space):

https://dialogic-docs.coppolaemilio.com/variables.html#22-conditions
https://dialogic-docs.coppolaemilio.com/timeline-text-syntax.html#about-special-events
https://dialogic-docs.coppolaemilio.com/creating-extensions.html#7-custom-settings-pages

When editing with VScode you can see a dot where is space  
![image](https://github.com/dialogic-godot/documentation/assets/9352894/97155554-b37d-4f0d-b9f7-a38c3713c827)  

I checked with python too  
![image](https://github.com/dialogic-godot/documentation/assets/9352894/578c1c39-94be-4271-b0aa-2906e1991ee7)  

It seems to be replaced by the mistake in #34

**Comment by hsandt** [comment]
Exactly, and when copying code snippets directly into GD scripts that causes an error:

> Parse Error: Invalid white space character U+00A0.

(try with the very first code snippet in https://docs.dialogic.pro/getting-started#5-adding-the-dialog-to-your-game)
